mitosheet/mitosheet/api/get_test_imports.py

```diff
@@ -17,12 +18,16 @@
 CSV_IMPORT_ERROR = 'There was an error importing this CSV file. Make sure that the file exists, or select a different file or dataframe.'
 EXCEL_IMPORT_ERROR = 'There was an error importing this sheet. Make sure that the Excel file exists and contains this sheet, or select a different file or dataframe.'
 DATAFRAME_IMPORT_ERROR = 'There was an error importing this dataframe. Make sure that the dataframe is defined, or select a different file or dataframe.'
+SNOWFLAKE_IMPORT_ERROR = 'There was an error executing this query. Make sure that your credentials are valid and you have access to this table, or select a different file or dataframe.'
```

Comment:
Figure out how to use this error message instead of the raised error message from the backend